core/controllers/editor.py

```diff
@@ -739,6 +687,7 @@ class StateYamlHandler(EditorHandler):
     layer.
     """
 
+    @acl_decorators.open_access
```

Comment:
No authorization was applied here. So I think it got left in the audit doc. But it uses user_id. So, should I allow open access or implement another decorator (It cannot use can_play_exploration decorator due to differing number of arguments)?